tests/test_xkls_calendar.py

```diff
@@ -89,6 +92,7 @@ def test_sunday_to_monday(self):
             pd.Timestamp('2014-09-01', tz=UTC),  # National Day
             pd.Timestamp('2012-09-17', tz=UTC),  # Malaysia Day
             pd.Timestamp('2016-12-26', tz=UTC),  # Christmas Day
+            pd.Timestamp('2020-05-25', tz=UTC),  # Eid al Fitr
```

Comment:
How is this passing? I don't see this change in the actual exchange calendar (but I see you mentioned it in the description).

Reply:
so we already had a '2020-05-24' (a Sunday) date listed for `malaysia_eid_al_fitr_first_day`, which has a sunday_to_monday observance. I just added this to the list of cases to test.